src/Controller/PressPacksController.php

```diff
@@ -109,10 +110,14 @@ public function pressPackAction(Request $request, string $id) : Response
                 return $parts;
             });
 
+        if ($this->isGranted('FEATURE_CAN_USE_HYPOTHESIS')) {
+            $arguments['hypothesisOpener'] = new HypothesisOpener();
+        }
+
         $arguments['relatedContent'] = $arguments['package']
-            ->then(Callback::methodEmptyOr('getRelatedContent', function (PressPackage $package) {
-                return ListingTeasers::basic($package->getRelatedContent()->map($this->willConvertTo(Teaser::class, ['variant' => 'secondary']))->toArray());
-            }));
+              ->then(Callback::methodEmptyOr('getRelatedContent', function (PressPackage $package) {
```

Comment:
strnge indentation, method calls usually follow 4 spaces like everything else. `php-cs-fixer` doesn't catch this, or the build would have failed?

Reply:
Will fix.

I ran proofreader locally to fix style issues before pushing, and this got through. Code sniffer config may need a tweak?